.github/workflows/makefile.yaml

```diff
@@ -22,41 +22,38 @@ jobs:
       packages: read
       actions: read
     steps:
-    -
-      name: Checkout repository
-      uses: actions/checkout@v4
-      id: git
-      with:
-        submodules: 'recursive'
-    -
-      name: Unshallow clone for tags
-      id: tags
-      run: |
-        sudo chown -R $(whoami) /__w/pulumi-provider-boilerplate/pulumi-provider-boilerplate
-        git config --global --add safe.directory /__w/pulumi-provider-boilerplate/pulumi-provider-boilerplate
-        git fetch --prune --unshallow --tags
-    -
-      name: Build
-      id: build
-      run: |
-        make build
-    -
-      name: Install
-      id: install
-      run: |
-        set -ex
-        make install
-    -
-      name: PulumiUp
-      id: up
-      run: make up
-    -
-      name: PulumiDown
-      id: down
-      run: make down
-    -
-      name: Generate multi-language examples from yaml IaC program
-      id: examples
-      run: |
-        set -ex
-        make gen_examples
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        id: git
+        with:
+          submodules: "recursive"
+      - name: Unshallow clone for tags
+        id: tags
+        run: |
+          sudo chown -R $(whoami) /__w/pulumi-provider-boilerplate/pulumi-provider-boilerplate
+          git config --global --add safe.directory /__w/pulumi-provider-boilerplate/pulumi-provider-boilerplate
+          git fetch --prune --unshallow --tags
+      - name: Setup tools
+        uses: ./.github/actions/setup-tools
+        with:
+          tools: go, pulumicli, pulumictl, nodejs, python, dotnet
+      - name: Build
+        id: build
+        run: |
+          make build
+      - name: Install
+        id: install
+        run: |
+          set -ex
+          make install
+      - name: PulumiUp
```

Comment:
I'm a bit confused by this and it looks broken to me. This is the `up` directive from the makefile:

```
up::
	$(call pulumi_login) \
	cd ${EXAMPLES_DIR} && \
	pulumi stack init dev && \
	pulumi stack select dev && \
	pulumi config set name dev && \
	pulumi up -y
```

It looks like it runs `pulumi up` from the `examples repo, but that is not a pulumi program, as there are subfolders for each language and each example. Am I missing something?

Reply:
Not sure. I've not changed this - just reformated the file while fixing the outdated go version.

Reply:
Oh I think this is fine because it's actually using the yaml example:

```makefile
EXAMPLES_DIR    := ${WORKING_DIR}/examples/yaml
```